src/odemis/driver/technolution.py

```diff
@@ -175,21 +175,21 @@ def __init__(self, name, role, host, children, externalStorage, **kwargs):
             ckwargs = children["EBeamScanner"]
         except Exception:
             raise ValueError("Required child EBeamScanner not provided")
-        self._ebeam_scanner = EBeamScanner(parent=self, **ckwargs)
+        self._ebeam_scanner = EBeamScanner(parent=self, daemon=kwargs.get("daemon"), **ckwargs)
```

Comment:
Changes made to to the technolution wrapper to have the yaml startup the wrapper correctly.